_templates/component/new/Component.module.css.t

```diff
@@ -0,0 +1,7 @@
+---
+to: src/components/<%= h.changeCase.pascal(name) %>/<%= h.changeCase.pascal(name) %>.css
```

Comment:
is the `module` extension missing here?

Reply:
I wasn't entirely sure I wanted to add that much "noise" to the file if it already is setup to work without it. Worth having a discussion, though

Reply:
it's just a convention for when using CSS modules, files without the `.module.` extension are considered global styles

Reply:
Yeah, that is true. I'm happy to make the switch.